### ISSUE TITLE
Awkward Looking Bears

### ISSUE THREAD

**Issue by cordy-123** [jasonrohrer/OneLife#932]
I'm guessing, it's from this change https://github.com/jasonrohrer/OneLifeData7/commit/06e2334a86c756576e5ba63037e91325498a42d8

But now, at bear caves that are blocked by unpassable item, the bear just very awkwardly stays there on the bear cave tile. 
I'm guessing the transition to return 'AWAKE' bears to caves was accidentally removed?

Used to be if you tap a Bear Cake with 'Waking' Bear and if the cave is blocked by unpassable item, the bear would return back to it's cave within a second.

![image](https://github.com/jasonrohrer/OneLife/assets/61180955/781913be-3539-4452-96b1-f04e24eb47b3)
![image](https://github.com/jasonrohrer/OneLife/assets/61180955/561ba0b3-a6a1-48c8-b165-cfdf06bd0a43)

![image](https://github.com/jasonrohrer/OneLife/assets/61180955/7faf2e55-cb1e-4e46-89de-90e93252ccaa)

**Comment by NotSoShady116** [comment]
Idk they look ok to me

**Comment by jasonrohrer** [comment]
Yeah, I guess they should go back in eventually, if they can't get out... I'll think about how to fix this.
